Makefile

```diff
@@ -20,7 +21,7 @@ list_sources:
 
 build:
 	mkdir -p ${BUILD_DIR}
-	nvcc -x cu ${SOURCE_FILES} --std=c++11 -lineinfo -o ${BUILD_DIR}/${EXEC_FILE} -Xcompiler -openmp
+	nvcc -x cu ${SOURCE_FILES} --std=c++11 -lineinfo -o ${BUILD_DIR}/${EXEC_FILE} -Xcompiler -fopenmp
```

Comment:
`-fopenmp` flag is throwing alot of warnings when building on Windows. Looks like compiler differences. `-fopenmp` is connected with gcc, windows nvcc command supports only cl.exe. It will be good to differentiate this flag based upon OS (like in SOURCE_FILE initialization).